File: src/server/Models/NationSimplified.cs
namespace Models
{
    public class NationSimplified
    {
        public int NationId { get; set; }
        public string NationName { get; set; }
        public string RulerName { get; set; }
        public string AllianceName { get; set; }

        public NationSimplified() { }

        public NationSimplified(int nationId, string nationName, string rulerName, string alliance)
        {
            NationId = nationId;
            NationName = nationName;
            RulerName = rulerName;
            AllianceName = alliance;
        }
    }
}

File: src/client/src/app/models/nationSimplified.ts
export class NationSimplified {
    nationId: number;
    nationName: string;
    rulerName: string;
    allianceName: string;

    constructor(nationId: number = 0, nationName: string = "", rulerName: string = "", allianceName: string = "") {
        this.nationId = nationId;
        this.nationName = nationName;
        this.rulerName = rulerName;
        this.allianceName = allianceName;
    }
}